Automattic/vip-go-mu-plugins
# Add Environment helpers requirement in 000-pre-vip-config

## Description

This PR adds a require to lib/helpers/environments, thus exposing `vip_get_env_var` early. 

## Changelog Description

### Load vip_get_env_var early

We've implemented early loading of `vip_get_env_var` so that it's available in vip-config.php

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Checkout the PR
2. Add `require_once __DIR__ .  '/wp-content/mu-plugins/000-pre-vip-config/requires.php';` somewhere on your local install prior to loading `vip-config.php`
3. Verify that the helper function works as expected.
